e2e/provision/playbooks/roles/install/defaults/main.yml

```diff
@@ -26,6 +26,9 @@ nephio_stock_repos:
   - name: free5gc-packages
     repo: https://github.com/nephio-project/free5gc-packages.git
     branch: main
+  - name: catalog
+    repo: https://github.com/nephio-project/catalog.git
+    branch: main
 
```

Comment:
For OAI the packages are in `oai` repository --> https://github.com/OPENAIRINTERFACE/oai-packages/tree/r2 right now it is in `r2` branch. I will move them to `main` once we have done the integration. 

However, for R2 I was thinking of installing the packages in the readme. This way the users will understand how to onboard a package. In R1 we were not showing it.

Reply:
Ok, I can also include that repository to the list. The `catalog` repository is going to be for the [OAI Nephio network package](https://github.com/nephio-project/catalog/pull/7)

Reply:
I meant about the other repository -->  https://github.com/OPENAIRINTERFACE/oai-packages/tree/r2 which contains only core network packages.

Reply:
I just included as part of the list ;)